docs/customer-interactive-api.md

```diff
@@ -0,0 +1,60 @@
+# Livery Video Interactive API
+
+This document describes the API calls that are available to customers. These calls can be used by application administrators to retrieve restricted data.
+
+For more information regarding domain & paths, see: **Livery Video Customer API**
+
+## Pull restricted broadcast data
+
+Authorized customers are allowed to access Livery interactive restricted API calls.
+Authorization will be done via an API key that is configured by a supervisor.
```

Comment:
Authorization will be done via an API key that can be retrieved from the Livery interactive portal.